cognite/client/_api/sequences.py

```diff
@@ -858,7 +839,7 @@ def list(
         return self._list(list_cls=SequenceList, resource_cls=Sequence, method="POST", filter=filter, limit=limit)
 
 
-class SequencesDataAPI(APIClient):
+class SequencesRowsAPI(APIClient):
```

Comment:
Renaming like this makes for a lot  unnecessary noise in the PR. Could it be done in another PR?

Reply:
Sure.